### ISSUE TITLE
File organization

### ISSUE THREAD

**Issue by sashahafner** [Chih-yuHung/STM-and-1D-model#1]
@Chih-yuHung should we have one directory for 1D model, one for STM, and later then a separate directory for comparison that could pull output from the other two?

**Comment by Chih-yuHung** [comment]
Yes! it's a great idea. It would be easier for me to have two directories. After we have the results, a separate directory can be created to store the results.

**Comment by sashahafner** [comment]
Great. I will add a directory for STM stuff. Do you want to move the existing files into a 1D directory? Otherwise I can, but maybe some are meant to be shared files.

**Comment by Chih-yuHung** [comment]
I moved the .R files to a 1D directory.
